bench/gen_synthetic_dune_watch.ml

```diff
@@ -21,10 +21,10 @@ let write_subset base_dir library_index subset =
             List.flatten
               (List.map
                  (fun k ->
-                   List.map
-                     (fun j ->
-                       sprintf "M_%d_%d_%d_%d.f()" (library_index - 1) j mod_rows k)
-                     (count subsets_per_library))
+                    List.map
```

Comment:
I like that the function bodies are indented by two from the `fun` and not from the `(` in front of `fun`. Seems more consistent.